<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAjoutLivreWidget</class>
 <widget class="QWidget" name="QAjoutLivreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAjoutLivreWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="LineTitre"/>
   </item>
   <item row="7" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Titre :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QTextEdit" name="TextResume">
     <property name="minimumSize">
      <size>
       <width>652</width>
       <height>141</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Auteur :</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Tags :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="LineAuteur"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>141</height>
      </size>
     </property>
     <property name="text">
      <string>Résumé :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="3">
    <widget class="QPushButton" name="BtnValider">
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="TextError">
     <property name="styleSheet">
      <string notr="true">QLabel{
color : rgb(255, 73, 73);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>287</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="BoxDivers">
           <property name="text">
            <string>Divers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxPolicier">
           <property name="text">
            <string>Policier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxAventure">
           <property name="text">
            <string>Aventure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxAction">
           <property name="text">
            <string>Action</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxEnfants">
           <property name="text">
            <string>Enfants</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxSciFi">
           <property name="text">
            <string>Science-Fiction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxScience">
           <property name="text">
            <string>Sciences</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxPolitique">
           <property name="text">
            <string>Politique</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxHistoire">
           <property name="text">
            <string>Histoire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BoxFantastique">
           <property name="text">
            <string>Fantastique</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>BtnValider</sender>
   <signal>clicked()</signal>
   <receiver>QAjoutLivreWidget</receiver>
   <slot>Valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Valider()</slot>
 </slots>
</ui>
